R/assertions.R

```diff
@@ -832,9 +853,6 @@ assert_list_of <- function(arg, class, named = FALSE, optional = TRUE) {
 #'
 #' Assert that all elements of the argument are named.
 #'
-#' @param message string passed to `cli_abort(message)`.
-#' When `NULL`, default messaging is used.
-#' `"{arg_name}"` and `"{indices}"` can be used in messaging.
```

Comment:
Was this removed intentionally?
Now `"{indices}"` is no longer mentioned in the documentation.

Reply:
Yes, removed intentionally because it's no longer always available (which allows each of the checks within the function to have more independent messaging per your request)

Reply:
But there is only one check within the function. And if `message` is used `indices` is available. Or did I miss something?

Reply:
Yes, for `assert_list_of()` but not for `assert_named()`. The section was removed from the `assert_named()` documentation.

Reply:
There are plenty of internal objects we're not documenting available in certain cases. I like the consistent documentation across the functions

Reply:
OK